### ISSUE TITLE
Kingmaking

### ISSUE THREAD

**Issue by millerj-tc** [ACG8/Lucid#13]
One problem that seems inherent in the game right now is that you can lose because another player built a bad deck. If you're playing in a game against a fast thief deck you may have cards that slow them down for your slower build, but they can still hit 20 off of another player who just declined to include enough good traps/barriers. This is another reason to finish the round before counting points so that if you're neck and neck with someone and then they go first and dive someone who misplayed their scry you don't lose because of the other player's misplay.

I guess with the idea of the security deck we just need to make the security deck strong enough against thief aggro that the super linear/fast thief deck gets super hosed if anyone is playing security cards?

This is also mitigated by allowing substantial "sideboarding" between matches -- changing out cards to deal with what their opponents are doing. Or even just bringing multiple decks?

**Comment by ACG8** [comment]
I don't really see a way around this without compromising the interactivity of the game. Kingmaker is always a problem in multiplayer games, and if players can choose their targets you will always have this issue. I agree about finishing the round before counting points; since points can change after the game ends anyway, I see no reason to deny some players the same number of turns as other players.

**Comment by millerj-tc** [comment]
This is a super big picture solution/idea, but I was also thinking:

Imagine this gets published and popular and stores want to run Lucid events. What if we explain in our marketing materials and game manuals that the only sanctioned Lucid events are "community events"  rather than "tournaments." We explain that the purpose of randomly matching people at a CE is to give people a chance to meet new players.

We encourage people to be friendly and to always include someone new in their game they haven't played with before if they're playing casually at a store (much easier in a multiplayer format since you can play with friends and a new person simultaneously.)

We encourage stores to run weird formats where powerful in-meta cards are suddenly "house ruled" to be worth negative starting VP so that people have to bring in new decks, rather than just relying on always playing the same stuff.

Basically, if we can't stop kingmaking, we try and influence around the social interactions of the game instead. So that when a less experience player gives away a win, the other players offer to help them in a friendly way or just shrug it off rather than getting peeved at them.

**Comment by ACG8** [comment]
That sounds good. I agree that it is pretty big picture, though. For the time being, I think we should mostly just ignore the kingmaker problem (as many games do).

**Comment by millerj-tc** [comment]
Added a note at the beginning of the rulebook about Spirit of Community, based on Ultimates's Spirit of the Game rule.
